content/docs/command-reference/remove.md

```diff
@@ -8,15 +8,16 @@ Remove DVC-tracked files or directories from the <abbr>workspace</abbr>.
 usage: dvc remove [-h] [-q | -v] [-o | -p] [-f] targets [targets ...]
 
 positional arguments:
-  targets        DVC-files to remove.
+  targets        stages (found in `dvc.yaml`) or `.dvc` files to remove.
 ```
 
 ## Description
 
 This command safely removes data files or directories that are tracked by DVC
-from the <abbr>workspace</abbr>. It takes a
-[DVC-File](/doc/user-guide/dvc-file-format) as input, removes all of its outputs
-(`outs`), and optionally removes the DVC-file itself.
+from the <abbr>workspace</abbr>. It takes a stage name (see -n option of dvc
+run) or a .dvc file as target, removes all of its outputs (outs field), and
+optionally removes the stage entry in [dvc.yaml(/doc/user-guide/dvc-file-format)
+or the .dvc file itself.
```

Comment:
Always surround command names in back quotes please. `` `dvc run` ``. Also missed link for .dvc file".

Reply:
Also the last .dvc is missing back quotes. Please be careful when copy-pasting to make sure you've reviewed all text. Thanks

Reply:
I guess I can just fix it now so we can move on:
```suggestion
from the <abbr>workspace</abbr>. It takes a stage name (see -n option of
`dvc run`) or a .dvc file as target, removes all of its outputs (outs field), and
optionally removes the stage entry in [dvc.yaml(/doc/user-guide/dvc-file-format)
or the `.dvc` file itself.
```